swagger.json

```diff
@@ -1860,19 +1388,23 @@
                     "type": "string"
                 },
                 "maxSpeed": {
-                    "type": "number"
+                    "type": "number",
+                    "description": "in knots"
                 },
                 "averageSpeed": {
-                    "type": "number"
+                    "type": "number",
+                    "description": "in knots"
                 },
                 "distance": {
-                    "type": "number"
+                    "type": "number",
+                    "description": "in meters"
                 },
                 "duration": {
                     "type": "integer"
                 },
                 "startTime": {
-                    "type": "string"
+                    "type": "string",
+                    "format": "date-time"
```

Comment:
Is there a way to provide some date time samples or be more specific about format? I get a lot of question about the format.